vcf/filter.go

```diff
@@ -174,27 +208,36 @@ func FilterVcfPos(vcfs []*Vcf) []*Vcf {
 	chrVcfMap := make(map[string][]*Vcf)
 
 	var ref []dna.Base
-	var alt []dna.Base
+	var alt [][]dna.Base
+	var containsN bool
 	for _, v := range vcfs {
 		chrVcfMap[v.Chr] = append(chrVcfMap[v.Chr], v)
 	}
 	var i int
 	var curr []*Vcf
 	for key := range chrVcfMap {
 		curr = chrVcfMap[key]
-		encountered := make(map[int64]bool)
+		encountered := make(map[int]bool)
 		for i = 0; i < len(curr); i++ {
 			if encountered[curr[i].Pos] == true {
 				//do not add
 			} else {
 				encountered[curr[i].Pos] = true
+				containsN = false
 				ref = dna.StringToBases(curr[i].Ref)
-				alt = dna.StringToBases(curr[i].Alt)
-				if dna.CountBaseInterval(ref, dna.N, 0, len(ref)) == 0 && dna.CountBaseInterval(alt, dna.N, 0, len(alt)) == 0 {
+				alt = GetAltBases(curr[i].Alt)
```

Comment:
Not for now, but this seems to be similar code to what I saw in a function above.  I bet we could reduce code by having a ContainsN(vcf) bool
function that would check for Ns and give a true/false.